packages/acceptance-tests/pkg-tests-specs/sources/commands/_entry.test.js

```diff
@@ -0,0 +1,26 @@
+describe(`Entry`, () => {
+  describe(`version option`, () => {
+
+    // regex credit: https://github.com/semver/semver/issues/232
```

Comment:
@arcanis let me know if this is overkill

Reply:
Probably 😄 If we do that better use `semver.valid()`